recipes/drflac/all/conanfile.py

```diff
@@ -10,15 +13,16 @@ class DrflacConan(ConanFile):
     topics = ("audio", "flac", "sound")
     license = ("Unlicense", "MIT-0")
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "compiler", "build_type", "arch"
+    settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False], 
         "fPIC": [True, False],
         "buffer_size": "ANY",
```

Comment:
`"ANY"` appears to be no longer valid for Conan v2. What would be the equivalent for it?

Reply:
@chausner `["ANY"]` is the correct approach for Conan v2: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#any-special-value

Reply:
Fixed in 7d4cdc453539e07d646adbfb9cf0350427ed16f3